release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -1560,3 +1560,26 @@ advisory. See the following article for notes on the container images in this
 release:
 
 link:https://access.redhat.com/solutions/4758151[{product-title} 4.3.0 container image list]
+
+[[ocp-4-3-1]]
+=== RHBA-2020:0390 - {product-title} 4.3.1 Bug Fix Update
+
+Issued: 2020-02-10
+
+{product-title} release 4.3.1 is now available. The list of packages included in
+the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2020:0390[RHBA-2020:0390] advisory.
+The container images and bug fixes included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2020:0391[RHBA-2020:0391] advisory.
+
+Space precluded documenting all of the container images for this release in the
+advisory. See the following article for notes on the container images in this
+release:
+
+link:https://access.redhat.com/solutions/4796501[{product-title} 4.3.1 container image list]
+
+[[ocp-4-3-1-upgrading]]
+==== Upgrading
+
+To upgrade an existing {product-title} 4.3 cluster to this latest release, see
+xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI for instructions.]
```

Comment:
After this sentence on upgrading, this is the IMPORTANT note to add (commented out for now):

Note that this will need to be carried forward in future 4.3.z RNs as well.

```
[IMPORTANT]
====
If you are upgrading to this release from {product-title} 4.2 or 4.3.0, you must restart all Pods after the upgrade is complete.

This is because the service CA is automatically rotated as of {product-title} 4.3.1. The service CA is rotated during the upgrade and a restart is required afterward to ensure that all services are using the new service CA before the previous service CA expires.

After this one-time manual restart, subsequent upgrades and rotations will ensure restart before the service CA expires without requiring manual intervention.
====
```
This note is being added several places throughout the documentation through this PR: https://github.com/openshift/openshift-docs/pull/19246

Reply:
Just added the commented out note